backend/pkg/api/handlers/common.go

```diff
@@ -158,6 +159,10 @@ func (v *validationError) checkPassword(password string) string {
 	return v.checkRegex(rePassword, password, "password")
 }
 
+func (v *validationError) checkConfirmationHash(hash string) string {
+	return v.checkRegex(reEmailConfirmationHash, hash, "token")
+}
```

Comment:
Question: you have made sure that new tokens match this regex?

Reply:
Yes should match, at least unless the generation charset (that function was taken from v1) is changed https://github.com/gobitfly/beaconchain/blob/staging/backend/pkg/commons/utils/user.go#L31